docker/images/circle/Dockerfile

```diff
@@ -0,0 +1,28 @@
+ARG QEMU_PLATFORM
+FROM balenalib/${QEMU_PLATFORM}-debian-golang
+# FIXME: If not repeated here build fails.
+# See https://travis-ci.org/simelo/libskycoin/jobs/529481211#L649-L653
+
+ARG GITHUB_OAUTH_TOKEN
+ARG CIRCLE_PROJECT_USERNAME
```

Comment:
Remove `CIRCLE` prefix . Has to be generic , reusable in many services .

Reply:
Okay, good idea